### DISCUSSION TITLE
GitHub Sponsors Referral Link

### DISCUSSION THREAD

**Discussion by RhysSullivan** [community/community#48671] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi!

I develop https://www.answeroverflow.com/ which helps developers answer questions on Discord for different open source programming projects. I'd like to implement a way for developers who are answering questions to link their GitHub sponsors account when the question asker marks their question as solved

Demo here: https://twitter.com/RhysSullivan/status/1629999682041659394

Is there any way / any plans to have referral codes on a sponsor link, i.e https://github.com/sponsors/sapphiredev?ref=answeroverflow ? If this gets considered I'd love to see statistics on the number of people who use the referral link and also possibly have some sort of kickback from GitHub for encouraging these users to sponsor on GitHub

It's got the following benefits:

- Supports developers who are answering questions
- Encourages visibility of GitHub sponsors
- Supports customers integrating GitHub sponsors into their applications

Thanks for considering this!

**Comment by RhysSullivan** [comment]
Slight bump on this if that's ok, thanks!

**Reply by xobazovv97-hash** [reply]
????

**Comment by Kenxpx** [comment]
GitHub Sponsors does not support referral links or tracking.
There’s currently no way to see referrals or get kickbacks from sponsor links.
